### DISCUSSION TITLE
Display Psalm version on https://psalm.dev/ sandbox

### DISCUSSION THREAD

**Discussion by alies-dev** [vimeo/psalm#8648] [Ideas]
There is no Psalm version info on https://psalm.dev/

<img width="1126" alt="image" src="https://user-images.githubusercontent.com/5278175/199485403-285a3223-913d-4423-adca-35c1fff38aa4.png">

Version info is important to know when to test code-snippet before submitting in issue to this repository

**Comment by danog** [comment]
The version is present in the expanded mode:
![image](https://user-images.githubusercontent.com/7339644/199486587-528415d6-fedc-4895-b78a-03be116f2e7d.png)

**Reply by alies-dev** [reply]
thanks! That's pretty good, better to see version number + commit hash, but I can live with it :)
